miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -55,6 +57,9 @@ open class MiniAppMessageBridge {
     private var allowScreenOrientation = false
     private lateinit var miniAppSecureStorageDispatcher: MiniAppSecureStorageDispatcher
 
+    /** provide MiniAppCloseAlertInfo to HostApp to show close alert popup. */
+    var miniAppShouldClose: MiniAppCloseAlertInfo? = null
```

Comment:
I think we can make this private and set a getter function for host app to get this value, so that host app can't manipulate the actual value comes from js. what do you think @khairul-alam-licon ?